index.html

```diff
@@ -183,12 +231,31 @@ <h6 id="new-feature-title" class="rest"></h6>
                         <i class="iconfont icon-caret-down onlyRest"></i>
                     </a>
                     <div class="dropdown-menu" aria-labelledby="rest-time-onlyrest-presets-dropdown">
-                        <a class="dropdown-item" href="javascript:setPresetTime('rest-time-onlyrest', 3, '3分钟')" onKeyDown="this.click()" onKeyPress="this.click()" onKeyUp="return false;">3分钟</a>
-                        <a class="dropdown-item" href="javascript:setPresetTime('rest-time-onlyrest', 5, '5分钟')" onKeyDown="this.click()" onKeyPress="this.click()" onKeyUp="return false;">5分钟</a>
-                        <a class="dropdown-item" href="javascript:setPresetTime('rest-time-onlyrest', 10, '10分钟')" onKeyDown="this.click()" onKeyPress="this.click()" onKeyUp="return false;">10分钟</a>
-                        <a class="dropdown-item" href="javascript:setPresetTime('rest-time-onlyrest', 15, '15分钟')" onKeyDown="this.click()" onKeyPress="this.click()" onKeyUp="return false;">15分钟</a>
-                        <a class="dropdown-item" href="javascript:setPresetTime('rest-time-onlyrest', 20, '20分钟')" onKeyDown="this.click()" onKeyPress="this.click()" onKeyUp="return false;">20分钟</a>
-                        <a class="dropdown-item" href="javascript:setPresetTime('rest-time-onlyrest', 30, '30分钟')" onKeyDown="this.click()" onKeyPress="this.click()" onKeyUp="return false;">30分钟</a>
+                        <script>
+                            // Populate rest-time-onlyrest preset dropdown items
+                            const restOnlyrestPresets = [3, 5, 10, 15, 20, 30];
+                            restOnlyrestPresets.forEach(function(preset) {
+                                // Get the correct localized minutes text
+                                let minutes = i18n.__('minutes');
+                                
+                                // For Chinese languages, use specific characters
+                                const currentLang = store.get('i18n') || 'en';
+                                if (currentLang === 'zh-CN') {
+                                    minutes = '分钟';
```

Comment:
Hard-coded strings like this is not preferred and does not support fourth locale support even if there is not such a thing as for now.